bql/semantic/expression.go

```diff
@@ -129,54 +145,59 @@ func (e *evaluationNode) Evaluate(r table.Row) (bool, error) {
 	case GT:
 		return csEL > csER, nil
 	default:
-		return false, fmt.Errorf("boolean evaluation require a boolen operation; found %q instead", e.op)
+		return false, fmt.Errorf("boolean evaluation requires a boolean operation; found %q instead", e.op)
 	}
 }
 
-// cellFromRow retrives the value of a binding from a given row
+// cellFromRow retrives the value of a binding from a given row.
 func cellFromRow(binding string, r table.Row) (*table.Cell, error) {
 	var (
 		val *table.Cell
 		ok  bool
 	)
 	val, ok = r[binding]
 	if !ok {
-		return nil, fmt.Errorf("comparison operations require the binding value for %q for row %q to exist", binding, r)
+		return nil, fmt.Errorf("comparison operation requires the binding value for %q for row %q to exist", binding, r)
 	}
 	return val, nil
 }
 
 // comparisonForLiteral represents the internal representation of an expression of comparison between a binding and a literal.
 type comparisonForLiteral struct {
-	op OP // operation
+	op OP // operation.
 
-	lS string // left string
-	rS string // right string
+	lS string // left string.
+	rS string // right string.
 }
 
 func (e *comparisonForLiteral) Evaluate(r table.Row) (bool, error) {
 	leftCell, err := cellFromRow(e.lS, r)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("in comparisonForLiteral.Evaluate got error: %v", err)
```

Comment:
Similar comment: you got error extracting a cell from a row, right? What were you trying to extract? From where? Might these help the person debugging this to find the issue, along with the error message?

Reply:
Done.